update_cache.sh

```diff
@@ -57,6 +57,10 @@ for arg in "$@"; do
 		clone_if_not_present WikibaseLocalMedia.git https://github.com/ProfessionalWiki/WikibaseLocalMedia.git
 		fetch_all WikibaseLocalMedia.git
 
+		# WikibaseEdtf
+		clone_if_not_present WikibaseEdtf.git https://github.com/ProfessionalWiki/WikibaseEdtf
```

Comment:
(super minor nitpick) I’d prefer a .git at the end of the URL just like above

Reply:
done